web/assets/css/webdashboard.css

```diff
@@ -728,3 +728,56 @@ table th {
         width: 200px;
     }
 }
+
+/* Rules for the Graph displayed on the locale view */
+#graphdiv {
+    width: 700px;
+    height: 350px;
+    margin-left: -20px;
```

Comment:
The graph title right now is really near the graph area. I'd put this to `-10px`and add `top: -10px;` to `.dygraph-title`.